setup.py

```diff
@@ -31,7 +31,7 @@
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
     python_requires=">=3.7",
-    install_requires=["tutor>=14.0.0,<15.0.0"],
+    install_requires=["tutor>=15.0.0,<16.0.0"],
```

Comment:
maybe it'd easier for future migrations to use just `install_requires=["tutor"],`

Reply:
I think this comes from the change in [codejail](https://github.com/eduNEXT/tutor-contrib-codejail/pull/2). This is the way of pinning the version used across almost all plugins.